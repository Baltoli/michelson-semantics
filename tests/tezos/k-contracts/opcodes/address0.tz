$contract { parameter (contract unit);
storage (option address);
code {CAR; ADDRESS; SOME; NIL operation; PAIR }
 } ;
$param "tz1cxcwwnzENRdhe2Kb8ZdTrdNy4bFNyScx5" ;
$storage None ;
$ExpectedOutput (Some "tz1cxcwwnzENRdhe2Kb8ZdTrdNy4bFNyScx5") ;

