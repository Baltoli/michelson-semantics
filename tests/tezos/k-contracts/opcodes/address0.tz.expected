<generatedTop>
  <k>
    .
  </k>
  <stack>
    Pair .List Some #Address ( "tz1cxcwwnzENRdhe2Kb8ZdTrdNy4bFNyScx5" )
  </stack>
  <paramtype>
    contract .AnnotationList unit .AnnotationList
  </paramtype>
  <storagetype>
    option .AnnotationList address .AnnotationList
  </storagetype>
  <balance>
    #Mutez ( 0 )
  </balance>
  <amount>
    #Mutez ( 0 )
  </amount>
  <now>
    #Timestamp ( -1 )
  </now>
  <myaddr>
    #Address ( "InvalidMyAddr" )
  </myaddr>
  <knownaddrs>
    .Map
  </knownaddrs>
  <sourceaddr>
    #Address ( "InvalidSourceAddr" )
  </sourceaddr>
  <senderaddr>
    #Address ( "InvalidSenderAddr" )
  </senderaddr>
  <chainid>
    #ChainId ( 0 )
  </chainid>
</generatedTop>
